src/main/java/Lesson3/Task2/Plane.java

```diff
@@ -0,0 +1,17 @@
+package Lesson3.Task2;
+
+class Plane extends Vehicle {
+    int height;
+    int passPlaces;
+    int year;
```

Comment:
You need to add only Plane specific fields here (height and passPlaces), all the common are inherited from Vehicle